src/main/java/frc/robot/Constants.java

```diff
@@ -14,5 +14,5 @@
  */
 public final class Constants {
 
-   public static final Double MAX_SPEED = 0.8;
+  public static final Double MAX_SPEED = 0.8;
```

Comment:
When we do the rewrite lets make sure we make sure that we have all parameters are constants and organize them into relevant constant files.